### ISSUE TITLE
DAX Studio Connecting Issue

### ISSUE THREAD

**Issue by karthick159** [DaxStudio/DaxStudio#607]
HI,

When I try to connect live power Bi Dataset to DAX studio i am getting below error. Please suggest any Idea 

![Dax studio Connecting issue](https://user-images.githubusercontent.com/83665637/117047537-9f24f680-ad2f-11eb-98b9-b2834fd69237.JPG)

![image](https://user-images.githubusercontent.com/83665637/117047561-a8ae5e80-ad2f-11eb-8a0c-e53ac63ee9ea.png)

**Comment by dgosbell** [comment]
This looks like an error from the dataset. Are you able to connect with other tools like SSMS or even Power BI Desktop?

**Comment by karthick159** [comment]
I am able to connect the dataset from the workspace in POWER BI desktop

**Comment by dgosbell** [comment]
So have you re-tried DAX Studio since trying PBI Desktop? The error message sounds like it could be transient. I've never seen the service throw that sort of error before and I've reached out to Microsoft to see if I can get any more information about it because it does not happen for me.

**Comment by dgosbell** [comment]
Did you re-test with DAX Studio at the same time as testing with Power BI Desktop?

According to the product team you would get this error if someone attempted to move a model using the large model storage option to a different region. So maybe whoever attempted to do this has now undone this change and fixed the issue

**Comment by karthick159** [comment]
Yes i have retried in Dax studio but i am getting the same error but i can access the workspace through Powerbi Dasktop.

**Comment by dgosbell** [comment]
Sorry, but I have no idea what is causing this and I cannot reproduce this issue. That error is not one generated by DAX Studio, it's coming from the Power BI Service and according to the product team it should only happen if you have attempted to move a model between regions. 

You might have to try raising a support ticket with Microsoft for this. They should be able to trace that RootActivityId to get more of an idea of what is going on here.

**Comment by karthick159** [comment]
Thanks for your time . I will reach out POWER BI support team
